CovertInfiltration/Src/CovertInfiltration/Classes/UICovertActionsGeoscape.uc

```diff
@@ -1126,8 +1126,35 @@ simulated function bool OnUnrealCommand(int cmd, int arg)
 
 simulated function OnRemoved()
 {
+	local XComGameStateHistory History;
+	local XComGameState NewGameState;
+	local XComGameState_CovertAction ActionState;
+	local bool bModified;
+
 	super.OnRemoved();
 
+	// Turn off the "Now Available" flag for any new covert actions
+	History = `XCOMHISTORY;
+	NewGameState = class'XComGameStateContext_ChangeContainer'.static.CreateChangeState("Turn off new Covert Action Now Available flag");
+	foreach arrActions(ActionState)
+	{
+		if( ActionState.bNewAction && ActionState.CanActionBeDisplayed())
```

Comment:
`ActionState.CanActionBeDisplayed()` is a redundant check - actions that cannot be displayed will never end up in `arrActions`